benchmarks/src/main/scala/monix/connect/benchmarks/redis/RedisBenchFixture.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020-2020 by The Monix Connect Project Developers.
+ * See the project homepage at: https://connect.monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.benchmarks.redis
+
+import io.lettuce.core.RedisClient
+import io.lettuce.core.api.StatefulRedisConnection
+import monix.connect.redis.Redis
+import monix.execution.Scheduler.Implicits.global
+
+trait RedisBenchFixture {
+  final val redisUrl = "redis://localhost:6379"
+  implicit val connection: StatefulRedisConnection[String, String] = RedisClient.create(redisUrl).connect()
+
+  val maxKey: Int = 5000
+  val maxField: Int = 500
+  val maxValue: Int = 50000
+
+  var key: Int = 0
+  var field: Int = 0
+  var value: Int = 0
```

Comment:
I think it would be better not to use mutable variables here, if we do run this benchmark in parallel, threads could use the same value multiple times... 

Would be better to replace the `getNextKey` and `getNextField` by just creationg an array of range x on the set up `[1, 2, 3, 4, ..., X]` that would be used from the tests?

Then for the values we could just insert random values or even the same one every time (replacing `genNextValue`)